docs/pages/admin-guides/management/security/reduce-blast-radius.mdx

```diff
@@ -13,28 +13,12 @@ Teleport encourages users to practice defense in depth so that every component o
 - [Set up your RBAC without admin roles](#set-up-your-rbac-without-admin-roles)
 - (!docs/pages/includes/tctl.mdx!)
 
-## Make MFA mandatory for `tsh login`
+## Use MFA for local user authentication
 
-If a user sets up an account to authenticate to their Teleport cluster with only a password, an adversary can gain access to the password using brute-force attacks, person-in-the-middle attacks, or phishing. But even if a user's password is compromised, you can stop an attacker from authenticating with it when they run `tsh login`.
+For local users Teleport requires using a MFA device or One-time password (OTP) when they register their user. An OTP is considered a less secure option
+as they are easier to copy and not physical devices.  With MFA Teleport allows for further options of allowing or denying device models. 
```

Comment:
I don't love this intro. The grammar feels off and the justification for OTP being less secure feels weak.